nss-day-cohort-46/rare-rest-rare-api-cool-cake
# Pp get post comments

Details:
Backend for getting a post's comments is complete.

Testing Instructions:
Postman URL:
```
http://localhost:8000/comments/1
```

Make a GET request and see if it returns all of the comments for that post.
